apps/demo/src/locales/fr/user.json

```
@@ -50,6 +50,23 @@
     },
     "title": "Forget Password (fr)"
   },
+  "invitation": {
+    "form": {
+      "actions": {
+        "cancel": "Cancel (fr)",
+        "submit": "Invite (fr)"
+      },
+      "email": {
+        "label": "Email (fr)",
+        "placeholder": "john.doe.at.example.com"
+      },
+      "role": {
+        "label": "Role (fr)",
+        "placeholder": "Select role (fr)"
+      }
+    },
```

```suggestion
    },
    "modal":{
      "button":{
        "label":"Invite (fr)"
      },
      "dialog":{
        "header":"Invitation form (fr)"
      }
    },
```
Add modal related translations too